platform/platform-app/src/main/resources/features.xml

```diff
@@ -26,108 +26,136 @@
 <features name="${project.artifactId}-${project.version}">
 
     <!--
-    Begin Camel Features 2.12.1; All feature definitions come from 3rd party feature file
-        (org.apache.camel.karaf/apache-camel/2.12.1/xml/features)
+    Begin Camel Features 2.16.1; All feature definitions come from 3rd party feature file
+        (org.apache.camel.karaf/apache-camel/2.16.1/xml/features)
 
     DO NOT EDIT
     -->
     <!--We don't want this pulling in additional XML libraries on top of the ones already installed by CXF-->
-    <!--<feature name='xml-specs-api' version='2.2.0' resolver='(obr)' start-level='10'>-->
-    <!--<bundle dependency='true'>mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.1/2.2.0</bundle>-->
-    <!--<bundle dependency='true'>mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.stax-api-1.0/2.2.0</bundle>-->
-    <!--<bundle dependency='true'>mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.jaxb-api-2.2/2.2.0</bundle>-->
-    <!--<bundle dependency='true'>mvn:com.sun.xml.bind/jaxb-core/2.2.11</bundle>-->
-    <!--<bundle>mvn:org.codehaus.woodstox/stax2-api/3.1.4</bundle>-->
-    <!--<bundle>mvn:org.codehaus.woodstox/woodstox-core-asl/4.4.1</bundle>-->
+    <!--<feature name="xml-specs-api" version="2.5.0" resolver="(obr)" start-level="10">-->
```

Comment:
Yeah, I was leaving them here as a reference since this is the feature copied out of Camel's features file. This is just so you don't have to look up the camel features file in order to see what exactly was in there if we need to change it down the road.